charts/openfga/templates/rbac.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
The role binding is needed if the workload is being run under a service account. The role binding allows the `wait-for-migration` init container to poll the Kubernetes control plane to get the status of the migrate job.